hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmSnapshotManager.java

```diff
@@ -151,7 +151,9 @@ public static DBCheckpoint createOmSnapshotCheckpoint(
     // Write snapshot generation (latest sequence number) to compaction log.
     // This will be used for DAG reconstruction as snapshotGeneration.
     dbCpDiffer.appendSequenceNumberToCompactionLog(dbLatestSequenceNumber,
```

Comment:
We can rename this method (and the comment above) now that it does more than just appending sequence number. Like `appendSnapshotCreationToCompactionLog`

Reply:
+1.